eds_scikit/io/arrow.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+import pandas as pd
+import pyarrow as pa
+
+
+class arrowConnector:  # pragma: no cover
+    def __init__(self, sql, database_name: str):
+        self.database_name = database_name
+        self.db = (
+            sql(f"DESCRIBE DATABASE EXTENDED {database_name}")
+            .filter("database_description_item=='Location'")
+            .collect()[0]
+            .database_description_value
+        )
+
+    def get_pd_fragment(
+        self,
+        table_name: str,
+    ):
+
+        self.path_table = os.path.join(self.db, table_name)
+
+        # Import the parquet as ParquetDataset
+        parquet_ds = pa.parquet.ParquetDataset(
+            self.path_table, use_legacy_dataset=False
+        )
+
+        # Partitions of ds
+        fragments = iter(parquet_ds.pieces)
```

Comment:
`pieces` is deprecated, we should use fragments instead. Also, `iter` doesn't look necessary here since fragments is a list.
```suggestion
        fragments = parquet_ds.fragments
```

Reply:
But `fragments` isn't available on pyarrow 0.17.0, so I think we should stock to `pieces` for now ?

Reply:
Ah yes you're right I was using the latest version, let's keep it simple and use `pieces` then.